### PR TITLE
Goal emojis

### PR DESCRIPTION
**Pull request by Agill-Sheron**
https://linear.app/geyser/issue/GYS-7843/goals-emojis

### CODE REVIEW

File: src/modules/project/pages/projectView/views/projectMainBody/components/GoalModal.tsx [hunk 1]
```diff
@@ -10,10 +10,14 @@ import {
   Text,
   VStack,
 } from '@chakra-ui/react'
+import EmojiPicker from 'emoji-picker-react'
+import { Categories } from 'emoji-picker-react'
```
Comment (sajald77) [inline src/modules/project/pages/projectView/views/projectMainBody/components/GoalModal.tsx:14]: build failing due to unused import

**Review by sajald77 — COMMENTED** [review]
Looks good, is this done? 
I've only added a build failing comment, looks good.

File: src/modules/project/pages/projectView/views/projectMainBody/components/GoalModal.tsx (hunk 1 above)
Comment (Agill-Sheron) [reply src/modules/project/pages/projectView/views/projectMainBody/components/GoalModal.tsx]: Yes it's done, we decided not to go with the fluentui emojis that will take more efforts to implement

File: src/modules/project/pages/projectView/hooks/useProjectGoalForm.tsx [hunk 2]
```diff
@@ -78,13 +79,15 @@ export const useProjectGoalForm = (
 
   useEffect(() => {
     if (goal) {
+      console.log(goal)
```
Comment (sajald77) [inline src/modules/project/pages/projectView/hooks/useProjectGoalForm.tsx:82]: remove console log

File: src/modules/project/pages/projectView/views/projectMainBody/components/GoalModal.tsx [hunk 3]
```diff
@@ -41,19 +44,30 @@ const denominationOptions = [
...
-  const { control, handleSubmit, loading, watch, errors, enableSubmit } = useProjectGoalForm(
+  const { control, handleSubmit, loading, watch, errors, enableSubmit, setValue } = useProjectGoalForm(
     goal || null,
     projectId,
     onClose,
     refetch,
   )
   const isCompleted = goal && goal.status === ProjectGoalStatus.Completed
 
+  const [isEmojiPickerOpen, setIsEmojiPickerOpen] = useState(false)
```
Comment (sajald77) [inline src/modules/project/pages/projectView/views/projectMainBody/components/GoalModal.tsx:55]: for true false, scenario, you can simple use, useDisclosure, it give you {isOpen, onOpen, onClose, onToggle}. You don't have to handle it manually. It's okay this way too.

**Review by sajald77 — APPROVED** [review]
Looks, good added a couple of suggestions, try to get to those if you have time before you merge.

**Pull request merged**